### DISCUSSION TITLE
【求助】【失物招领】有没有人丢了一个笔记本呀！

### DISCUSSION THREAD

**Discussion by w1ntercube** [share121/inter-knot#2499] [其他]
![DeathNote](https://github.com/user-attachments/assets/12f47658-ef61-43a3-8e3b-085f59e015b2)

今天放学在六分街捡到一个笔记本，看样子还挺新的，也没有写名字，我发到绳网上看看有没有人认领>.<

**Comment by cpu644** [comment]
你可以写几个名字进去。

**Comment by Bai6869** [comment]
听说写自己的名字有奇效<img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/8481df2cd99d2b00ecb94a92a14c9fe4_7915718047845434249.png" alt="米游兔 OK" width="75" >

**Comment by Lixia256** [comment]
如果你的富豪太太哪天想要立遗嘱，你可以直接用这个笔记本上的纸打印，然后直接让对方签名就好了。

**Comment by hyperion-netizen** [comment]
先寫自己的名吧以免遺失（doge

**Comment by lllliufeng** [comment]
先写自己的名字宣誓主权（bushi
